Automattic/wp-calypso
# Migration trial: Update: upgrade plan screen

Closes #80174

## Proposed Changes

* Added logic to show the notification screen when a user is not eligible to try a free trial since it's already used

## Testing Instructions

* Log in as a non-a8c user (a8c user can create multiple sites with the TRIAL subscriptions)
* Go to `setup/import-focused/importerWordpress?siteSlug={SIMPLE_SITE_SLUG}&from={JN_SITE_URL }&option=everything`
* Check if "Try it for free" button has a popover message on hover/focus
<img width="805" alt="Screenshot 2023-08-14 at 13 52 41" src="https://github.com/Automattic/wp-calypso/assets/1241413/e86a07d4-e226-4965-926d-d289b2dedfff">

* Go to `setup/import-focused/migrationTrial?siteSlug={SIMPLE_SITE_SLUG}&from={JN_SITE_URL }&option=everything`
* Check if the "You already have an active free trial" screen is rendered
* Check if the "Purchase and migrate" button leads to the checkout page

**NOTE:** I would keep this screen if someone goes directly to the **migrationTrial** route

<img width="784" alt="Screenshot 2023-08-09 at 22 48 47" src="https://github.com/Automattic/wp-calypso/assets/1241413/2ca3fa80-afb1-4ef8-aa27-72358cb08617">


## Pre-merge Checklist
